test/cases/testdata/refheads/test-regressions.yaml

```diff
@@ -113,4 +113,42 @@ cases:
     p[{"a": "b"}] = true
   query: data.test.p[{"a":"b"}] = x
   want_result:
-  - x: true
\ No newline at end of file
+  - x: true
+- note: regression/full extent with non-string (number) last term
+  modules:
+  - |
+    package test
+    p[0] = true
```

Comment:
Not a fan of `:=`, are you? ;)

Reply:
lol I look at it like the compiler does. 😜